python/tests/experimental/core/test_udf_schema.py

```diff
@@ -46,11 +46,11 @@ def test_udf_pandas() -> None:
 
 @register_dataset_udf(["col1"])
 def add5(x):
```

Comment:
```suggestion
def add5(x: Union[List, Dict[str, Any]]) -> Union[List, pd.Series]:
```
Type hint UDF signatures